src/app/__tests__/App.test.js

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
```

Comment:
This incorrect change is back! Please change the test back to test App. Hint: `git checkout origin/master -- path/to/file`

Reply:
This should be back now!